### PR TITLE
chore(router): inherit more app proxy settings from v1.x

### PR DESCRIPTION
**Pull request by krancour**
Fill out the remaining proxy settings for an app, minus anything having to do with SSL, enforcing HTTPS, etc.

### CODE REVIEW

File: nginx/config.go [hunk 1]
```diff
@@ -65,8 +70,16 @@ http {
 		listen 80{{ if $routerConfig.UseProxyProtocol }} proxy_protocol{{ end }};
 		server_name {{$domain}};
 		{{ if $appConfig.Available }}location / {
+			proxy_buffering off;
+			proxy_set_header Host $host;
+			proxy_set_header X-Forwarded-For {{ if $routerConfig.UseProxyProtocol }}$proxy_protocol_addr{{ else }}$proxy_add_x_forwarded_for{{ end }};
+			proxy_redirect off;
+			proxy_connect_timeout 30s;
```
Comment (helgi) [inline nginx/config.go]: This is configurable in 1.x - might as well keep parity here

Comment (helgi) [inline nginx/config.go]: Just noticed it isn't for the k8s side but is for any other situation - I'm kinda confused why that is. Maybe someone else knows the legacy reason for that

File: nginx/config.go [hunk 2]
```diff
@@ -65,8 +70,16 @@ http {
...
 			proxy_send_timeout {{ $routerConfig.DefaultTimeout }}s;
 			proxy_read_timeout {{ $routerConfig.DefaultTimeout }}s;
```
Comment (helgi) [inline nginx/config.go]: These two are configurable independently in 1.x when not on k8s but are exactly like this under k8s. Curious.

Comment (krancour) [inline nginx/config.go]: Interesting... if that is indeed a mistake, it should be fixed in a separate PR.  This one doesn't introduce it.  I will look into it further.

**Comment by helgi** [comment]
All looks good from my side of it

File: nginx/config.go (hunk 2 above)
Comment (krancour) [inline nginx/config.go]: Ah... disregarding this due to offline convo.

**Pull request merged**